src/routes/+page.svelte

```diff
@@ -220,10 +220,10 @@
     {@html opengraph_html}
 </svelte:head>
 
-<home-container class="block md:p-[1.25vw] md:pr-[3.75vw] mt-16 md:mt-0">
+<home-container class="mt-16 block md:mt-0 md:p-[1.25vw] md:pr-[3.75vw]">
     <hero-section class="flex flex-col justify-between md:flex-row">
         <latest-animes-slider
-            class="relative h-[22.5rem] w-full md:h-[27.875vw] md:w-[42.1875vw]"
+            class="relative h-96 w-full md:h-[27.875vw] md:w-[42.1875vw]"
```

Comment:
This is not good. We are looking to be dynamic

Reply:
Yea, I will look into this later